src/commands/config/create.ts

```diff
@@ -0,0 +1,180 @@
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as CreatedWalletPath } from "../../implementations/wallet/create";
+import { getLogger } from "../../logger";
+import { highlight } from "../../utils";
+import { handler as CreateTemporaryDns } from "../dns/txt-record/create";
+import ConfigTemplate from "./config-template.json";
+import { CreateConfigCommand } from "./config.type";
+
+interface ConfigFile {
+  wallet: string;
+  forms: Form[];
+}
+
+interface Form {
+  defaults: {
+    issuers: (VerifiableDocumentIssuers | TransferableRecordIssuers)[];
+  };
+}
+interface VerifiableDocumentIssuers {
+  id?: string;
+  revocation?: { type: string };
+  name: string;
+  documentStore?: string;
+  identityProof: {
+    type: string;
+    location?: string;
+    key?: string;
+  };
+}
+interface TransferableRecordIssuers {
+  name: string;
+  tokenRegistry: string;
+  identityProof: {
+    type: string;
+    location?: string;
+  };
+}
+
+const { trace } = getLogger("config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
```

Comment:
actually why this option ? why not create directly withing output-dir ?

Reply:
i think there is a typo, do you mean within or without?

within:
craeted wallet would be inside this output-dir as well. But for the case when users are supplying their already created wallet, they would include this option.

without:
thought it would be easier for all the files (config and wallet) created to be saved in the 'output-dir' folder.

let me know what do you think.